web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -425,6 +425,10 @@ $.extend(fixmystreet.set_up, {
                 $category_meta.empty();
             }
         });
+
+        if (fixmystreet.hooks) {
```

Comment:
This should be `if (fixmystreet.hooks && fixmystreet.hooks.update_problem_fields)` in case `hooks` later on gets defined elsewhere.